File: python/create-model.py
import requests

url = "https://app.nanonets.com/api/v2/ObjectDetection/Model/"

payload = "{\"categories\" : [\"TieFighter\", \"MillenniumFalcon\"]}"
headers = {
    'Content-Type': "application/json",
    }

response = requests.request("POST", url, headers=headers, auth=requests.auth.HTTPBasicAuth('REPLACE_API_KEY', ''), data=payload)

print(response.text)

File: node/create-model.js
const request = require('request');
const api ='Enter-your-api-key'

const data = JSON.stringify({
  "categories": [
    "TieFighter",
    "MillenniumFalcon"
  ]
});


const resp = request.post({
    url: `https://app.nanonets.com/api/v2/ObjectDetection/Model/`,
    data,
    auth: {
        username: api,
        password: ''
    }
}, (err, body) => {
    callback(err, body)
});